ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/Document.java

```diff
@@ -27,6 +27,16 @@ public class Document {
   @JsonProperty("category_id")
   private String categoryId;
 
+  public Document(
```

Comment:
Can you add `@JsonCreator`. NFDIV had some complex issues with callbacks that were hard to replicate. Unfortunately I can't remember the exact details but I think it was not deserializing complex types inside an unwrapped complex type correctly.